packages/material-ui/src/Dialog/Dialog.test.js

```diff
@@ -84,133 +93,118 @@ describe('<Dialog />', () => {
     assert.strictEqual(wrapper.hasClass('woofDialog'), true);
   });
 
-  it('should render Fade > div > Paper > children inside the Modal', () => {
```

Comment:
That test isn't very useful since there a many other intermediary components. This test compounds many other test anyway and can be safely removed.